dask/dataframe/core.py

```diff
@@ -577,6 +577,40 @@ def _aca_agg(self, token, func, aggfunc=None):
         """ Wrapper for aggregations """
         raise NotImplementedError
 
+    def rolling(self, window, min_periods=None, win_type=None):
+        """Provides rolling transformations.
+
+        Parameters
+        ----------
+
+        window: int
+            Size of the moving window. This window must be smaller
+            than the size of the previous partition.
+        min_periods: int, default None
+            Minimum number of observations in window required to have
+            a value (otherwise result is NA).
+        win_type: string, default None
+            Provide a window type. (Identical to pandas.)
+
+        The center, freq, and axis arguments are not supported.
+        """
+        from dask.dataframe.rolling import Rolling
+
+        if not isinstance(window, int):
+            raise TypeError('window must be an integer')
+        if window < 0:
+            raise ValueError('window must be positive')
+
+        if min_periods is not None:
+            if not isinstance(min_periods, int):
+                raise TypeError('min_periods must be an integer')
+            if min_periods < 0:
+                raise ValueError('min_periods must be positive')
+
+        return Rolling(self, {'window': window,
```

Comment:
I left the old code untouched, figuring that would guarantee 0 change in behavior and would let the pandas deprecation warning bubble up, but it's obviously easy enough to do so. I don't know dask's policies with respect to deprecated features in pandas.